docs/source/distributed.md

```diff
@@ -217,7 +217,7 @@ inconsistent 'UUID' assignment across ranks, and to prevent races during initial
 ```
 
 ```{eval-rst}
-.. autofunction:: torch.distributed.distributed_c10d.is_xccl_available
+.. autofunction:: is_xccl_available
```

Comment:
```suggestion
.. autofunction:: torch.distributed.distributed_c10d.is_xccl_available
```

Reply:
This is user facing documentation and users can just call  `torch.distributed.is_xccl_available()`. Other backends have similar documentation:

- https://docs.pytorch.org/docs/stable/distributed.html#torch.distributed.is_nccl_available
- https://docs.pytorch.org/docs/stable/distributed.html#torch.distributed.is_gloo_available

instead of the full one.